internal/const/oceanbase/oceanbase.go

```diff
@@ -24,7 +27,24 @@ const (
 )
 
 const (
-	DefaultDiskUsePercent = 90
+	DefaultDiskExpandPercent  = 10
+	DefaultLogPercent         = 80
+	InitialDataDiskUsePercent = 20
+	DefaultDiskUsePercent     = 95
+	DefaultMemoryLimitPercent = 90
+)
+
+const (
+	DefaultMemoryLimitSize  = "0M"
```

Comment:
Why are the default memorySize and datafileMaxSize `0M`?

Reply:
0M means it will be calculated by oceanbase automatically